### ISSUE TITLE
Contrast errors on login page

### ISSUE THREAD

**Issue by hashraf88** [Alfresco/alfresco-ng2-components#377]
To meet WCAG 2.0 AA standard contrast on images needs to be minimum 4.5:1.
Currently there are 17 errors 

Steps to reproduce:
1. Download Wave extension for Chrome
2. Launch Wave tool when on login page

![screen shot 2016-07-06 at 16 04 35](https://cloud.githubusercontent.com/assets/13200338/16622807/5e4086a8-4393-11e6-87b9-68605ae85145.png)

More information can be found at: 
https://www.w3.org/WAI/WCAG20/quickref/?showtechniques=143#visual-audio-contrast-contrast 
http://webaim.org/standards/wcag/checklist#sc1.4.3

**Comment by spotnape** [comment]
Really great to see that the new application is getting such a thorough accessibility assessment. Excellent!

**Comment by VitoAlbano** [comment]
to match the requirements for the contrast we need to change the colors of the login page.
here it is an example of how it should be to match the contrast requirements.

<img width="1438" alt="screen shot 2016-09-06 at 15 50 19" src="https://cloud.githubusercontent.com/assets/7974125/18278378/c679ddb4-7449-11e6-9f9d-ba48509841ca.png">

This needs to be approved by designers and @jsotiro.

**Comment by magemello** [comment]
Closing this issue, now we hide the header bar in the login page
